net.damonlynch.rapid_photo_downloader.desktop

```
@@ -0,0 +1,14 @@
+[Desktop Entry]
+Categories=Graphics;Photography;
+Comment=Download, rename and back up photos and videos from cameras and other devices
+Exec=rapid-photo-downloader %f
+GenericName=Photo Downloader
+Icon=net.damonlynch.rapid_photo_downloader.svg
```

this is icon *name* not file name.
```suggestion
Icon=net.damonlynch.rapid_photo_downloader
```

this file should be upstream anyway

---

I made the change thanks and for upstream i got some issue because the desktop file is not compilent 
https://github.com/damonlynch/rapid-photo-downloader/blob/main/data/net.damonlynch.rapid_photo_downloader.desktop.in